src/app/shared/search/search-filters/search-filter/search-facet-filter/search-facet-filter.component.ts

```diff
@@ -324,6 +275,67 @@ export class SearchFacetFilterComponent implements OnInit, OnDestroy {
     return getFacetValueForType(facet, this.filterConfig);
   }
 
+  protected retrieveFilterValues(useCachedVersionIfAvailable = true) {
+    const facetValues$ = observableCombineLatest([this.searchOptions$, this.currentPage]).pipe(
+      map(([options, page]) => {
+        return { options, page };
+      }),
+      switchMap(({ options, page }) => {
+        return this.searchService.getFacetValuesFor(this.filterConfig, page, options, null, useCachedVersionIfAvailable)
+          .pipe(
+            getFirstSucceededRemoteData(),
+            map((results) => {
+                return {
+                  values: observableOf(results),
+                  page: page
+                };
+              }
+            )
+          );
+      })
+    );
+
+    let filterValues = [];
+    this.subs.push(facetValues$.subscribe((facetOutcome) => {
```

Comment:
This `subscribe` shouldn't be here if this method is called from another `subscribe`

Reply:
done